web/war/src/main/webapp/js/detail/dropdowns/propertyForm/propForm.js

```
@@ -545,7 +546,6 @@ define([
                             metadata: this.currentMetadata
                         }, justification)
                 });
-                self.teardown();
```

Yes. This component is torn down elsewhere. Tearing down immediately after `addProperty` is triggered caused a silent failure.